README.md

```diff
@@ -473,11 +473,12 @@ Metadata flags:
 - :nerd_face: `--pidfile`: file path to write the task's pid. The CLI syntax conforms to Podman convention.
 
 Logging flags:
-- :whale: `--log-driver=(json-file)`: Logging driver for the container (default `json-file`).
+- :whale: `--log-driver=(json-file|journald)`: Logging driver for the container (default `json-file`).
```

Comment:
Could you update the shell completion too

Reply:
where ?

Reply:
fixed